src/tests/ftest/dfuse/pil4dfs_dcache.py

```diff
@@ -0,0 +1,360 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import re
+
+from command_utils import ExecutableCommand
+from command_utils_base import BasicParameter
+from dfuse_test_base import DfuseTestBase
+from exception_utils import CommandFailure
+from run_utils import run_remote
+
+
+class Pil4dfsDcacheCmd(ExecutableCommand):
```

Comment:
Will you please move this class definition to `dfuse_utils.py`? We try to define all command classes in a utility for re-use later.

Reply:
- [ ] Move class Pil4dfsDcacheCmd to dfuse_utils.py

Reply:
Fixed with commit eace57b3b479abd420dec07bf85cc46b4a1c9c31